paf-mvp-operator-client-express/src/operator-client-proxy.ts

```diff
@@ -130,6 +132,13 @@ export const addOperatorClientProxyEndpoints = (
     httpRedirect(res, url.toString(), 302);
   });
 
+  app.post(jsonProxyEndpoints.seed, cors(corsOptions), (req, res) => {
+    const request = JSON.parse(req.body as string) as PostSeedRequest;
+    const seed = client.buildSeed(request.transaction_ids, request.data);
+    const response = seed as PostSeedResponse; // For now, the response is only a Seed.
```

Comment:
Any reason for this comment? If so, would you add it to the comment?
My feeling is that the endpoint could return a simple seed forever, which is not an issue.

Reply:
No particular reason. 
For consistency with other endpoints, I prefer to have this mix of wrapper and DTO here.